docs/cvl/methods.md

```diff
@@ -203,58 +203,59 @@ methods {
-By default, the Prover will use an {ref}`AUTO summary <auto-summary>` for 
-unresolved function calls, but that may produce spurious counter examples. 
-The catch unresolved-calls entry lets the user refine the summary used for 
+By default, the Prover will use an {ref}`AUTO summary <auto-summary>` for
+unresolved function calls, but that may produce spurious counter examples.
+The catch unresolved-calls entry lets the user refine the summary used for
 unresolved function calls.
 
 ```{note}
-Only one catch unresolved-calls entry is allowed per a specification file. 
-When importing a specification with a catch unresolved-calls entry it will be 
+Only one catch unresolved-calls entry is allowed per a specification file.
+When importing a specification with a catch unresolved-calls entry it will be
 included as part of the current specification, and cannot be overridden.
 ```
 
-A catch unresolved-calls entry can only be summarized with a dispatch list 
-summary (and a dispatch list summary is only applicable for a catch 
-unresolved-calls entry). 
-A dispatch list summary directs the Prover to consider each of the methods 
-described in the list as possible candidates for this unresolved call. 
-The Prover will choose dynamically, that is, for each potential run of the 
-program, which of them to call. 
+A catch unresolved-calls entry can only be summarized with a dispatch list
+summary (and a dispatch list summary is only applicable for a catch
+unresolved-calls entry).
+A dispatch list summary directs the Prover to consider each of the methods
+described in the list as possible candidates for this unresolved call.
+The Prover will choose dynamically, that is, for each potential run of the
+program, which of them to call.
 It is done accurately by matching the selector from the call's arguments
 to that of the methods described in the dispatch list.
-If no method from the list matches, it will use the `default` summary, see 
+If no method from the list matches, it will use the `default` summary, see
 below.
 The dispatch list will contain a list of patterns and the default summary to use in case no function matched the selector.
 The possible patterns are:
-1. Exact function - a pattern specifying both a contract, and the 
+1. Exact function - a pattern specifying both a contract, and the
    function signature.
    Example: `C.foo(uint)`
-2. Wildcard contract - a pattern specifying the function signature to match 
+2. Wildcard contract - a pattern specifying the function signature to match
    this signature on all available contracts (including the primary contract).
    Example: `_.bar(address)`
-3. Wildcard function - a pattern specifying a contract, and matches all 
-   external functions in specified contract.
+3. Wildcard function - a pattern specifying a contract, and matches all
+   external functions in specified contract (This pattern will also include the
+   contract's fallback if it's implemented).
```

Comment:
This is the only change I could find. Next time please try to commit without the other line break changes.

Is it easy for you to revert? Then please only add what's required.

Reply:
yes this is the only change.

Reply:
I wasn't aware of this feature! Thanks for pointing it out to me.